### DISCUSSION TITLE
Upload complete but its not showing the 10.zip file in internet archive

### DISCUSSION THREAD

**Discussion by jackhallloween21** [sandeepyadav1478/Transfer-files-from-google-to-terabox-cloud#4] [General]
![image](https://user-images.githubusercontent.com/78194804/125408996-197d9100-e3d9-11eb-965c-50149b456417.png)
![image](https://user-images.githubusercontent.com/78194804/125409054-27331680-e3d9-11eb-887c-aeb3630c4bc4.png)
![image](https://user-images.githubusercontent.com/78194804/125409120-387c2300-e3d9-11eb-9611-7b09badc3d43.png)

**Comment by jackhallloween21** [comment]
![image](https://user-images.githubusercontent.com/78194804/125409673-c35d1d80-e3d9-11eb-8b6b-3b772000948e.png)
 i didn't ran the last node. after running it my file is now visible

**Comment by sandeepyadav1478** [comment]
get download link of 10.zip and upload it to terabox
